3-probability/06-normal-approx/notes.qmd

```diff
@@ -0,0 +1,841 @@
+---
+title: "Continuous Distributions and Normal Approximations"
+subtitle: "Connections to boxes, continuous distributions, and a fundamental result"
+image: images/front-image.png
+format:
+  html: default
+  pdf: default
+editor_options: 
+  chunk_output_type: console
+---
+
+:::{.lo .content-hidden unless-profile="staff-site"}
+
+#### Concept Acquisition
+
+1. Connection with random variable and draws from a box
+2. IID random variables
+3. Sums and averages of IID random variables
+4. Continuous random variables and their pdf
+5. The unif(a,b) & normal distribution
+6. The Central limit theorem
+6. geom_density
+
+#### Tool Acquisition
+
+1. The empirical rule for a normal distribution
+2. pnorm, qnorm, rnorm, punif, qunif, runif
+
+#### Concept Application
+
+1. computing the EV and SE of a sum of iid random variables
+2. EV and SE for various distributions.
+3. Relationship to the box for discrete rvs
+4. Simulating box distributions and distributions of sums or averages of draws
+
+----------------------------------------------------
+
+:::
+
+
+```{r}
+#| echo: false
+#| message: false
+#| warning: false
+
+library(tidyverse)
+library(stat20data)
+library(infer)
+library(patchwork)
+```
+
+[I]{.dropcap}magine that a regular patron of a bar has hit the bottle rather hard one evening. When the bar closes for the night, they come out to weave their way home. Home is very near, in fact, just straight down the road. If our inebriate walks straight in the direction of their home, they can be there very soon. The only problem is - they can't walk straight. Every minute, they move in a random direction: backwards or forwards with equal probability. Where will they be after $n$ minutes? This is the famous "drunkard's walk" problem[^dw].
```

Comment:
I love this example, however, I'm wondering if its added complexity that's outside the scope of what we really need for this class.

"random walks", "sequences of random variables" - these are solidly topics from stochastic processes. It's always struck me as a higher order of complexity than vanilla intro prob, where you're just thinking about a single random variable. I think we always need to be on the lookout for scope-creep and work to keep our eye on the prize: asking, "what do we want them to learn and what is the minimal set of prereqs needed to get there"?

It's probably too late to revise anything here, but this could be marked for simplification in a future semester.

Reply:
You might be right. I wanted a funny  and interesting hook, and this seemed to be one that would incorporate both sums of rvs and clt ideas. But of course, it can be extremely complex and if they ask about when he would pass his home, it becomes out of scope.